lib/content-services/src/lib/ecm-user-info/ecm-user-info.component.html

```diff
@@ -0,0 +1,113 @@
+<div 
+    id="userinfo_container" 
+    [class.adf-userinfo-name-right]="showOnRight" 
+    (keyup)="onKeyPress($event)"
+    class="adf-userinfo-container" 
+    *ngIf="canShow"
+>
+    <span *ngIf="showName" id="adf-userinfo-ecm-name-display" class="adf-userinfo-name">
+        {{ecmUser | fullName}}
+    </span>
+    <button mat-button [matMenuTriggerFor]="menu" class="adf-userinfo-menu_button"
+            data-automation-id="adf-user-profile">
+        <div class="adf-userinfo-button-profile" id="user-profile">
+            <div *ngIf="identityUser; else showEcmUserImage" id="identity-user-image">
+                <div *ngIf="ecmUser?.avatarId; else initialTemplate">
+                    <div class="adf-userinfo-profile-container">
+                        <img id="logged-user-img" [src]="getEcmAvatar(ecmUser.avatarId)" alt="user-info-profile-button"
+                             class="adf-userinfo-profile-image"/>
+                    </div>
+                </div>
+                <ng-template #initialTemplate>
+                    <div [innerHTML]="identityUser | usernameInitials:'adf-userinfo-pic'"></div>
+                </ng-template>
+            </div>
+            <ng-template #showEcmUserImage>
+                <div id="ecm-user-image">
+                    <div *ngIf="ecmUser.avatarId; else initialTemplate" class="adf-userinfo-profile-container">
+                        <img id="logged-user-img" [src]="getEcmAvatar(ecmUser.avatarId)" alt="user-info-profile-button"
+                             class="adf-userinfo-profile-image"/>
+                    </div>
+                    <ng-template #initialTemplate>
+                        <div [outerHTML]="ecmUser | usernameInitials:'adf-userinfo-pic'"></div>
+                    </ng-template>
+                </div>
+            </ng-template>
+        </div>
+    </button>
+
+    <mat-menu #menu="matMenu" id="user-profile-lists" [xPosition]="menuPositionX" [yPosition]="menuPositionY"
+              [overlapTrigger]="false" class="adf-userinfo-menu">
+        <mat-tab-group id="tab-group-env" (click)="stopClosing($event)" selectedIndex="0" role="menuitem"
+                       class="adf-userinfo-tab adf-hide-tab">
+            <mat-tab id="ecm-panel" label="{{ 'USER_PROFILE.TAB.CS' | translate }}" role="dialog"
```

Comment:
not used anymore in the unit/e2e tests, from what I see

Reply:
Are you referring to the ecm-panel tab?

Reply:
yes, I checked the ADF, and no code references this id

Reply:
I will remove both references to this id (ecm and bpm components)